beautysh_sublime_plugin.py

```diff
@@ -0,0 +1,25 @@
+import sublime, sublime_plugin
+import subprocess
+
+class BeautyshCommand(sublime_plugin.TextCommand):
+	def run(self, edit):
+		p = subprocess.Popen(                                                  \
```

Comment:
I dislike how this is formatted, set the command array (`['python', ...]`) on a separate line, call it `beautysh_cmd` maybe, that should help make it smaller. Than just let autopep8 take care of it :)

Reply:
Addressed!

Reply:
Thanks!